client/imports/app/modules/core/csvtimeline/transactionComponent/transaction.component.ts

```diff
@@ -141,6 +152,54 @@ export class TransactionComponent implements OnInit, OnChanges {
 			this.filteradmin();
 		}
 	}
+	checkAlias(transactionData:any){
+		this.fillGstFormDataId=null;
+		console.log("callll",transactionData.Description)
+		this.gstData=Gst.find({transaction_id:transactionData._id}).fetch();
+		if(this.gstData && !this.gstData.length){
+				let descrption=transactionData.Description.toLowerCase();
+				_.forEach(Gst.find({}).fetch(),(value,key)=>{
```

Comment:
we should never use Gst.find({})  this will make live system crash. will work in dev because of less data